helmfile/charts/networkpolicy-workload/templates/ingress-nginx/default-backend.yaml

```
@@ -0,0 +1,41 @@
+{{ if .Values.ingressNginx.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: ingress-nginx-default-backend
+  namespace: ingress-nginx
+spec:
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/component: default-backend
+  policyTypes:
+    - Ingress
+    - Egress
+  ingress:
+    - from:
```

If you add this, then you could remove it later since it's the same ports, and you could also remove the namespace selector since it's in the same ns
```suggestion
    - from:
      - podSelector:
          matchLabels:
            app.kubernetes.io/component: controller
            app.kubernetes.io/instance: ingress-nginx
```